### PR TITLE
Completed Documentation for "pdftk" command

### LINKED ISSUE #452: Writing Documentation for "pdftk" command

### PR DESCRIPTION
**Pull request by alikanpurwala**
# Advanced HW 5 submission

Closes #452 

Name: Ali Kanpurwala
Uniqname: akanpurw

### CODE REVIEW

File: _commands/tools/pdftk.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<!-- minimal example -->
+~~~bash
+$ pdftk in1.pdf in2.pdf cat output combinedIn.pdf
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### 'pdftk locked.pdf input_pw YouShallPass output unlocked.pdf'
```
Comment (AmritHariharan) [inline _commands/tools/pdftk.md]: Don't just have the command as the title, have a short description of what it does, eg 'Decrypt PDF'

File: _commands/tools/pdftk.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+#### 'pdftk in1.pdf cat 1-4 12-end output out1.pdf'
+~~~bash
+$ pdftk in1.pdf cat 1-4 12-end output out1.pdf
+~~~
+
+This allows you to remove pages from a pdf. In this example pages 5 through 11 will be removed from the 'out1.pdf'
+
+#### 'pdftk in1.pdf cat 1east 2-end output out1.pdf'
+~~~bash
+$ pdftk in1.pdf cat 1east 2-end output out1.pdf
```
Comment (AmritHariharan) [inline _commands/tools/pdftk.md]: what flags would be used instead of `1east` to rotate over other angles?

**Comment by AmritHariharan** [comment]
Some minor changes for formatting/examples, have commented on the file. Also you may want to consider saying somewhere that the number ranges specified are which pages are _not_ being used (assuming that's actually how it works).

**Comment by alikanpurwala** [comment]
Updated Changes for the pdftk.md

**Comment by cyanliu** [comment]
You've been awarded credit for advanced homework 5 and given 2 points of extra credit for creating your initial PR more than 2 days before the deadline 🎉 

You won't see this immediately reflected on Canvas, but it will be there the next time we update grades!

**Pull request merged**